app/views/shared/_flash.html.erb

```diff
@@ -1,5 +1,5 @@
 <% flash.each do |type, msg| %>
-  <%= stylesheet_link_tag 'shared/flash' %>
+  <%= stylesheet_link_tag 'flash-message' %>
```

Comment:
I think you shouldn't paste `flash-message` file for each flash, it is ok to load it once

Reply:
If you don't want to load it unless you have any flashes, you can check it with `flash.any?`